devtools/projects/ng-devtools-backend/src/lib/client-event-subscribers.ts

```diff
@@ -210,17 +205,10 @@ const navigateRouteCallback = (messageBus: MessageBus<Events>) => (path: string)
 export const viewSourceFromRouter = (name: string, type: RoutePropertyType) => {
   const router: any = getRouterInstance();
 
-  if (router == null) {
+  if (router === null) {
     return;
   }
-
-  let element;
-  if (type === 'component') {
-    element = getComponentRefByName(router.config, name);
-  } else {
-    element = getElementRefByName(type, router.config, name);
-  }
-  return element;
+  return getRouterCallableConstructRef(router.config, type, name);
```

Comment:
Should we also rename it here ? 
```suggestion
  return getRouterCallableConstructRef(router.config, type, constructName);
```